lib/install/validate/validator.go

```diff
@@ -660,6 +660,7 @@ func (v *Validator) reachableRegistries(ctx context.Context, input *data.Data, p
 	if len(input.WhitelistRegistries) > 0 {
 		// ignoring error since default merge policy is union, so should never return
 		// an error
+		// #nosec: Errors unhandled.
 		m, _ := secureRegistriesSet.Merge(insecureRegistriesSet, nil)
```

Comment:
in theory, this error should be checked, although I checked the code for Merge, there is no any error returned right now, just has error in the interface.
but if in the future, if there is new struct introduced and start to return error, here will have problem.